composer.json

```diff
@@ -49,6 +46,13 @@
             "@phpunit-cover"
         ]
     },
+    "prefer-stable": true,
+    "repositories":[
```

Comment:
Do not forget to remove =)

Reply:
Yep, as soon as `avtocod/specs-php` will be released